src/__experimental__/components/select/option.component.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Option = ({ text, children, ...props }) => (
+  <div { ...props }>
```

Comment:
If `<Select>` uses a list tag then this should be an `<li>`. It should also have `role="option"` too to help with accessibility.